### PR TITLE
Added new helpers

### PR DESCRIPTION
**Pull request by dernster**
New helpers added:
- Views
  - `GradientView`
  - `OwnerView`
  - `RoundedView`, `RoundedImageView` and `RoundedButton`
- TableView
  - `func reloadDataAnimated(duration: NSTimeInterval = 0.4, completion: (() -> ())?)`
- Added ParametrizedString protocol
  - `func parametrize(parameters: CustomStringConvertible...) -> String`. For example: `"Hey {0}! It's been {1} years!".parametrize("Arnold", 3)` gives you `"Hey Arnlold! It's been 3 years!"`
  - `func parametrize(withDictonary dictonary: [Int: CustomStringConvertible]) -> String`

@xmartlabs/ios

### CODE REVIEW

File: Sources/ParametrizedString.swift [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+extension ParametrizedString {
+
+    private func replaceParameter(from string: String, index: Int, with stringConvertible: CustomStringConvertible) -> String {
+        let metaParameter = parameterFormat.stringByReplacingOccurrencesOfString("i", withString: "\(index)")
+        return string.stringByReplacingOccurrencesOfString(metaParameter, withString: stringConvertible.description)
+    }
+
+}
+
+extension String: StringProtocol {
```
Comment (mcousillas6) [inline Sources/ParametrizedString.swift:64]: This one should go below the protocol that implements

**Comment by mcousillas6** [comment]
👍

File: README.md [hunk 2]
```diff
@@ -114,6 +114,9 @@ This is a list of the helper functions and extensions implemented in this pod.
 * `func findFirstNumberInString() -> String?`: Returns the first number in a String if found
 * `func renderedHeightWithFont(font: UIFont, width: CGFloat) -> CGFloat`: Return the height necessary for a text given a width and font size. Same as `heightForString` extension on UIFont
 * `func getFirstAndLastName() -> (String,String)?`: Parses a first and a last name from a String. Takes last whitespace as separator for these values.
+* By conforming the `String` type to `ParametrizedString` protocol and specifying a parameter format like `"{i}"`:
+	* `func parametrize(parameters: CustomStringConvertible...) -> String`: Replace `"{i}"` substring with the i-th element of `parameters`. For example: `"Hey {0}! It's been {1} years!".parametrize("Arnold", 3)` gives you `"Hey Arnlold! It's been 3 years!"`
```
Comment (mats-claassen) [inline README.md]: Typo `Arnlold`

**Comment by mats-claassen** [comment]
🆗

**Pull request merged**